nautobot/docs/release-notes/version-2.1.md

```diff
@@ -38,6 +38,10 @@ Models with nullable fields (i.e. model fields with `null=True`) can now be filt
 !!! note
     Model fields that have the value `""` (i.e. blank) will not match with `__isnull=True`. Instead, they will match with `__isnull=False`.
 
+#### JSONVar ([#4926](https://github.com/nautobot/nautobot/issues/4926))
+
+Provides the ability to have native JSON data inputs for Jobs, this is provided by a multi-line text input on the Job form and the provided JSON data is serialized prior to passing to the `run()` method of the Job.
+
```

Comment:
- Perhaps changing the section heading to something like `Job JSONVar inputs` might add clarity?
- I wish we had a linter for this, but I'm trying to keep us sticking to the convention of previous releases of alphabetizing the headings under each major section - so this would better belong between `Job File Outputs` and `UI/API isnull Filter...` - would you mind making this change?

Reply:
Done